ios/adapters/epson/BLEPrinterEpsonAdapter.h

```diff
@@ -0,0 +1,26 @@
+//
+//  BLEPrinterEpsonAdapter.h
+//
+//  Created by Till POS on 14/09/21.
+//
+
+#pragma once
+#ifndef BLEPrinterEpsonAdapter_h
+#define BLEPrinterEpsonAdapter_h
+
+#if __has_include("RCTBridgeModule.h")
```

Comment:
what is this check for?

Reply:
Checks for the dependency if exists and then imports.
Like we did before .. similar code

Reply:
Then this logic should be replicated in NetPrinter also right?

Reply:
These files are same as before.
Might be some conditional imports are not needed.
But, I put the files same as they were before.